cmd/backup-manager/app/backup/backup.go

```diff
@@ -134,3 +135,128 @@ func constructOptions(backup *v1alpha1.Backup) ([]string, error) {
 	args = append(args, config.Options...)
 	return args, nil
 }
+
```

Comment:
clean implementation, if you have time better refactor backupData also, if not, leave a FIXME/TODO there

Reply:
ok, i think i can refactor backupData together.